test/jdk/jdk/internal/vm/Continuation/Fuzz.java

```diff
@@ -78,7 +78,8 @@ public class Fuzz implements Runnable {
     static final boolean RANDOM  = true;
     static final boolean VERBOSE = false;
 
-    static final int COMPILATION_TIMEOUT = 5_000; // ms
+    static float timeoutFactor = Float.parseFloat(System.getProperty("test.timeout.factor", "1.0"));
+    static final int COMPILATION_TIMEOUT = (int)(1_000 * timeoutFactor); // ms
```

Comment:
I was really hoping that `1_000 * timeoutFactor` would do the trick,
but stress run-#1 on my macosx-aarch64 machine had a couple of
sub-test timeouts at 12 seconds (1_000 * 12.0). My timeoutFactor
for slowdebug is 12.0. My linux-x64 machine passed stress run-#1
without any failures.

I've switched the value from 1_000 back to 5_000 for the next two
stress runs on both machines.